am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -88,6 +91,7 @@ public void grantExplicitResourceAccess(@NotNull @Valid ExplicitAccessGrant acce
                         .resourceName(accessGrant.getResourceName())
                         .attribute(attributePermission.getKey())
                         .securityClassification(accessGrant.getSecurityClassification())
```

Comment:
security classification should be dropped from explicit access?

Reply:
This is included in story 253 as far as I am aware and we are just removing the constraint for now. We checked this with katie.